soh/soh/SaveManager.cpp

```diff
@@ -1282,9 +1290,20 @@ void SaveManager::LoadBaseVersion3() {
         });
         SaveManager::Instance->LoadData("playTimer", gSaveContext.sohStats.playTimer);
         SaveManager::Instance->LoadData("pauseTimer", gSaveContext.sohStats.pauseTimer);
-        SaveManager::Instance->LoadArray("timestamps", ARRAY_COUNT(gSaveContext.sohStats.timestamp), [](size_t i) {
-            SaveManager::Instance->LoadData("", gSaveContext.sohStats.timestamp[i]);
+        SaveManager::Instance->LoadArray("itemTimestamps", ARRAY_COUNT(gSaveContext.sohStats.itemTimestamp), [](size_t i) {
+            SaveManager::Instance->LoadData("", gSaveContext.sohStats.itemTimestamp[i]);
         });
```

Comment:
So this change means that old timestamps will not be loaded into the new struct properly. Normally when a json change happens, we have to introduce a new LoadBase vX function. However this is non-critical information that will not break in-game functionality.

Curious others thought about this. do we want to add another base func and support the old timestamps to be loaded, or are we fine with old timestamps getting lost.

My gut feeling is I'm fine with losing the old timestamps. I think this is also probably necessary since the old timestamps included more than just "item" timestamps, so they probably wouldn't be able to be brought over correctly anyways without some strange parsing.

Reply:
I agree, losing timestamps if you load a save from an old build isn't a big enough problem to warrant building out migrations